source/read.txt

```diff
@@ -0,0 +1,130 @@
+.. _kotlin-sync-read:
+
+======================
+Read Data from MongoDB
+======================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :description: Learn how to use {+driver-short+} to read data from MongoDB.
+   :keywords: usage examples, query, find, code example
+
+.. toctree::
+   :titlesonly:
+   :maxdepth: 1
+
+   /read/specify-a-query
+   /read/retrieve
+   /read/project
+   /read/specify-documents-to-return
+   /read/count
+   /read/distinct
+   /read/cursors
+   /read/change-streams
+
+Overview
+--------
+
+On this page, you can see copyable code examples that show common
+methods you can use to retrieve documents with the {+driver-short+}.
+
+.. tip::
+
+  To learn more about any of the methods shown on this page, see the link
+  provided in each section.
+
+To use an example from this page, copy the code example into the
+:ref:`sample application <kotlin-sync-read-sample>` or your own application.
+Be sure to replace all placeholders in the code examples, such as ``<connection string URI>``, with
+the relevant values for your MongoDB deployment.
+
+.. _kotlin-sync-read-sample:
+
+.. include:: /includes/usage-examples/sample-app-intro.rst
+
+.. literalinclude:: /includes/usage-examples/sample-app.kt
+    :language: kotlin
+    :copyable:
+    :linenos:
+    :emphasize-lines: 19-21
+
+Find Documents
+--------------
```

Comment:
Note: Unlike most other driver, the Kotlin Sync driver doesn't have a strict `findOne()` method.